### PR TITLE
Edit the functions presentation

### PR DESCRIPTION
**Pull request by sleviim**
- Insert an image for understanding the function's structure
- Created an 'How to use' slide
- An exercise should be added later (TBD)

### CODE REVIEW

File: docs/course_materials/functions.html [hunk 1]
```diff
@@ -61,72 +57,91 @@ <h3><b>Functions</b></h3>
...
+
+say_hello("Joe") # Hello Joe
                     </code></pre>
                 </section>
+
                 <section>
                     <h3><b>Multiple Parameters</b></h3>
                     <ul>
-                        <li>A function can have any number of parameters</li>
+                        <li>A function can have more than one parameter</li>
```
Comment (rollandf) [inline docs/course_materials/functions.html:76]: A function can also have no parameters.

Comment (sleviim) [reply docs/course_materials/functions.html:76]: I'll change back to any

**Comment by rollandf** [comment]
Thanks!

:cookie:

**Comment by sleviim** [comment]
Freddy, can this one get merged?

**Comment by rollandf** [comment]
:dog2:

**Pull request merged**